static/js/deactivation-survey.js

```diff
@@ -109,15 +109,35 @@
         });
     }
 
-    const submitSurvey = async () => {
+    const submitSurvey = async (skipped = false) => {
         isSubmitting();
-        const surveyInput = document.getElementById('nfd-deactivation-survey__input').value;
+
+        let surveyInput = 'No input';
+        if (!skipped) {
+            inputValue = document.getElementById('nfd-deactivation-survey__input').value;
+            if (inputValue.length > 0) {
+                surveyInput = inputValue;
+            }
+        }
+
         // Send event
         const send = await sendEvent(surveyInput);
         deactivatePlugin();
     }
 
     const sendEvent = async (surveyInput) => {
+        let eventData = {
+            'label_key': 'survey_input',
+            'survey_input': surveyInput,
+            'category': 'user_action',
+            'brand': runtimeData.brand,
+            'page': window.location.href
+        }
+
+        if (getABTestPluginHome()) {
+            eventData.abTestPluginHome = true;
```

Comment:
@wpalani We do want to capture false values as well, not just true. The logic is, if the capability doesn't exist, don't do anything. If the capability exists, whether true or false, set that in the event data.

Reply:
Ok. Addresed in this commit: https://github.com/newfold-labs/wp-module-deactivation/pull/3/commits/f19e8f05bb00094e982e0d3bf4f021bb953ec2a6